foqus_lib/framework/optimizer/SM_Optimizer.py

```diff
@@ -57,13 +54,13 @@
     from pyomo.opt import SolverFactory
 
     pyutilib.subprocess.GlobalData.DEFINE_SIGNAL_HANDLERS_DEFAULT = False
-    from pyDOE import *
+    import pyDOE  # pylint: disable=unused-import
```

Comment:
Yes, this is a tricky one.  I left it in and added the pylint disable because I believe that Anuja (who wrote this code) was using it to detect if the SM Opt "plugin" was available or not.  I actually couldn't find where else it is used.  I would like to get @anujad95 to review and test this part directly, if possible.